userapi/storage/accounts/sqlite3/deltas/2022021012490600_add_account_type.go

```diff
@@ -0,0 +1,68 @@
+package deltas
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pressly/goose"
+
+	"github.com/matrix-org/dendrite/internal/sqlutil"
+)
+
+func init() {
+	goose.AddMigration(UpAddAccountType, DownAddAccountType)
+}
+
+func LoadAddAccountType(m *sqlutil.Migrations) {
+	m.AddMigration(UpAddAccountType, DownAddAccountType)
+}
+
+func UpAddAccountType(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+	ALTER TABLE account_accounts RENAME TO account_accounts_tmp;
+CREATE TABLE account_accounts (
```

Comment:
Why not just add the column? https://www.sqlite.org/lang_altertable.html

Reply:
I just copied the migration, I also thought there was something that prevented `ADD COLUMN`. Changed it now.